js_modules/dagster-ui/packages/ui-core/src/metadata/MetadataEntry.tsx

```diff
@@ -18,13 +18,16 @@ import {Link} from 'react-router-dom';
 import styled from 'styled-components';
 
 import {TableSchema} from './TableSchema';
-import {MetadataEntryFragment} from './types/MetadataEntryFragment.types';
+import {
+  MetadataEntryFragment,
+  TableMetadataEntryFragment,
```

Comment:
Importing the fragment instead of the `TableMetadataEntry` type allows us to avoid types that aren't included in the fragment.

Reply:
aha